README.md

```diff
@@ -39,6 +39,10 @@ To get the server running locally:
 | POST   | `/api/v1/user/signup`    | admin          | Create a new user account (staff and admin) | True                          |
 | POST   | `/api/v1/user/login`     | admin && staff | Login user (staff and admin)                | False                         |
 | POST   | `/api/v1/farmers/create` | admin && staff | Create new farmer                           | True                          |
+| GET    | `/api/v1/farmers` 				| admin && staff | Get All Farmers	                           | True                          |
+| GET    | `/api/v1/id`			 				| admin && staff | Get Farmer By Id	                           | True                          |
+
+
```

Comment:
Can you change the API route  from `/api/v1/id` to `/api/v1/farmers/:id`

Reply:
Done

Reply:
`farmers` and not `farmer`